Events/Ch24.event

```diff
@@ -0,0 +1,121 @@
+#include EAstdlib.event
+
+EventPointerTable(Chatper24Events,ThisChapter)
+#define Ch24GenLvlT1, 16 //level of generic unpromoted units
+#define Ch24GenLvlT2, 5 //level of generic promoted units
+#define CH24GenID, 0x71 //character id for generics
+ORG 0xE80000
+
+ThisChapter:
+POIN TurnBasedEvents
+POIN CharacterBasedEvents
+POIN LocationBasedEvents
+POIN MiscBasedEvents
+POIN Dunno Dunno Dunno
+POIN Tutorial
+POIN TrapData TrapData
+POIN Units Units
+POIN $0 $0 $0 $0 $0 $0
+POIN BeginningScene EndingScene
+
+Units:
+UNIT Eirika EphraimLord 0x00 Level(1, Ally, 0) [2,16] 0x00 0x00 0x0 0x00 [0x00] NoAI 
+UNIT Eirika EphraimLord 0x00 Level(1, Ally, 0) [1,17] 0x00 0x00 0x0 0x00 [0x00] NoAI 
+UNIT Eirika EphraimLord 0x00 Level(1, Ally, 0) [3,17] 0x00 0x00 0x0 0x00 [0x00] NoAI 
+UNIT Eirika EphraimLord 0x00 Level(1, Ally, 0) [0,18] 0x00 0x00 0x0 0x00 [0x00] NoAI 
+UNIT Eirika EphraimLord 0x00 Level(1, Ally, 0) [2,18] 0x00 0x00 0x0 0x00 [0x00] NoAI 
+UNIT Eirika EphraimLord 0x00 Level(1, Ally, 0) [4,18] 0x00 0x00 0x0 0x00 [0x00] NoAI 
+UNIT Eirika EphraimLord 0x00 Level(1, Ally, 0) [1,19] 0x00 0x00 0x0 0x00 [0x00] NoAI 
+UNIT Eirika EphraimLord 0x00 Level(1, Ally, 0) [0,20] 0x00 0x00 0x0 0x00 [0x00] NoAI 
+UNIT Eirika EphraimLord 0x00 Level(1, Ally, 0) [3,19] 0x00 0x00 0x0 0x00 [0x00] NoAI 
+UNIT Eirika EphraimLord 0x00 Level(1, Ally, 0) [2,20] 0x00 0x00 0x0 0x00 [0x00] NoAI 
+UNIT Eirika EphraimLord 0x00 Level(1, Ally, 0) [4,20] 0x00 0x00 0x0 0x00 [0x00] NoAI 
+UNIT Eirika EphraimLord 0x00 Level(1, Ally, 0) [5,19] 0x00 0x00 0x0 0x00 [0x00] NoAI 
+UNIT Eirika EphraimLord 0x00 Level(1, Ally, 0) [6,20] 0x00 0x00 0x0 0x00 [0x00] NoAI 
+UNIT
+
+
+TurnBasedEvents:
+TurnEventPlayer(0x0,EventAfterExitingPrepScreen,1)
+END_MAIN
+
+CharacterBasedEvents:
+END_MAIN
+
+LocationBasedEvents:
+Armory(Ch24Armory,0,0)

```

Comment:
Wait, so, if you go to the armory, you win (see area event)? Is this location correct?

Reply:
(Also on the AREA event, make it point to another event that checks the allegiance of the active unit. If it's not an ally, don't call the ending scene)